cli/azd/cmd/util.go

```diff
@@ -235,28 +224,28 @@ func ensureEnvironmentInitialized(ctx context.Context, environmentName string, e
 			if err != nil {
 				return fmt.Errorf("reading subscription id: %w", err)
 			}
-			if subscriptionSelection == "Other (enter manually)" {
+
+			if subscriptionSelection == manualSubscriptionEntryOption {
 				err = askOne(&survey.Input{
 					Message: "Enter an Azure Subscription to use:",
 				}, &subscriptionId)
 				if err != nil {
 					return fmt.Errorf("reading subscription id: %w", err)
 				}
 			} else {
-				subscriptionId = subscriptionSelection[len(subscriptionSelection)-len("(059cdffa-0e5b-47d8-ad4b-f13fd9099f21)")+1 : len(subscriptionSelection)-1]
+				subscriptionId = subscriptionSelection[len(subscriptionSelection)-len("(00000000-0000-0000-0000-000000000000)")+1 : len(subscriptionSelection)-1]
```

Comment:
Changed to "nil UUID" here to make more clear this UUID is just to ensure subscription ID is extracted correctly